### PR TITLE
Creating a new topics and resources area

### LINKED ISSUE #2303: Create new collections for resources and published articles
**Desired Behavior**
Need to create a few new collections in [layer5.io](https://layer5.io/) to hold published articles, which don’t fit into a [blog](https://layer5.io/blog) post, but are timeless.

Possible collections include `/service-mesh`, `/solutions`, `/resources`, `/videos` . New Wrapper style  and Graphql queries for these collections need to be created.

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #2303

**Notes for Reviewers**

New collection is available at `/topics`. It is a copy of the blog collection with only a couple of changes. More changes are needed to create something more like these:

- http://d2iq.com/resources
- http://datadoghq.com/resources
- https://www.nginx.com/resources

There is something broken with `/topics/tag/`. Help!

### CODE REVIEW

**Comment by leecalcote** [comment]
Tags don't seem to work for this new `topics` collection. 🤔

**Comment by l5io** [comment]
🚀 Preview for commit 576c93e787212d27ff547b86ee09512f000467e6 at: https://617cc9696961c5134972bc6a--layer5ng.netlify.app

File: src/sections/Topics/Topic-sidebar/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      </div>
+      
+      <div className="sidebar-widgets tags">
+        <div className="widgets-title">
+          <h3>Tags</h3>
+        </div>
+        <ul>
+          { tags && tags.map(tag => (
+            <li key={tag.fieldValue}>
+              <Link to={`/topics/tag/${slugify(tag.fieldValue)}`}>{tag.fieldValue} ({tag.totalCount})</Link>
```
Comment (debo19) [inline src/sections/Topics/Topic-sidebar/index.js:101]:
```suggestion
              <Link to={`/topic/tag/${slugify(tag.fieldValue)}`}>{tag.fieldValue} ({tag.totalCount})</Link>
```
@leecalcote the tags seem to be working with https://617cc9696961c5134972bc6a--layer5ng.netlify.app/topic/tag/envoy this URL 🤔

Comment (leecalcote) [reply src/sections/Topics/Topic-sidebar/index.js:101]: Aha! Thanks for figuring this out, ⚡️@debo19

**Comment by l5io** [comment]
🚀 Preview for commit 9c20a78236b4f55c7dc4f1d97b939dd7c64c969d at: https://617d5aac21a499e7b2a0581b--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@leecalcote good to merge? so that we can further work on this.

**Comment by leecalcote** [comment]
Yes, good call, @warunicorn19. Good to merge. 👍

**Review by warunicorn19 — APPROVED** [review]

**Pull request merged**